spark_log_parser/parsing_models/application_model_v2.py

```diff
@@ -241,7 +277,11 @@ def construct_spark_application(
         spark_app = self.compute_all_task_data(raw_data, spark_app)
         spark_app = self.compute_all_stage_data(raw_data, spark_app)
         spark_app = self.compute_all_driver_accum_data(raw_data, spark_app)
-        spark_app = self.compute_all_metadata(raw_data, spark_app)
+
+        metadata = self.compute_all_metadata(raw_data, spark_app)
+        if metadata is not None:
+            spark_app.metadata = metadata
```

Comment:
Is `None` not allowed?

Reply:
Probably not important either way... but I just did this so that we don't overwrite whatever default value may be assigned in the constructor